stdlib/3/asyncio/events.pyi

```diff
@@ -100,8 +100,8 @@ class AbstractEventLoop(metaclass=ABCMeta):
     @abstractmethod
     def call_soon_threadsafe(self, callback: Callable[..., Any], *args: Any) -> Handle: ...
     @abstractmethod
-    async def run_in_executor(self, executor: Any,
-                              func: Callable[..., _T], *args: Any) -> _T: ...
+    def run_in_executor(self, executor: Any,
```

Comment:
AbstractEventLoop still has an async def for `run_in_executor`. But maybe typeshed's tests complain if you keep it that way.

Reply:
Good catch!  Reverting the change on the AbstractEventLoop and seeing how the tests like it.

Reply:
Yea, mypy does not like the change in signature:

```console
error: Return type "Future[_T]" of "run_in_executor" incompatible with return type "Coroutine[Any, Any, _T]" in supertype "AbstractEventLoop"
```

I don't want to misdocument the base class; should I put a `# type: ignore` on my `BaseEventLoop` changes?  Can you even do that in .pyi files?

Reply:
Reverted the change to `AbstractEventLoop` and instead added an ignore statement with explanation.